src/pages/index.jsx

```diff
@@ -29,13 +29,16 @@ class Index extends React.Component {
 
   static renderCallouts() {
     return CALLOUTS.map((card) => {
-      const cardClass = `card home-callout col-md-4 callout--${card.BADGE}`;
+      const cardClass = `card col-home-callout home-callout callout--${card.BADGE}`;
       return (
         <div className={cardClass} key={card.TITLE}>
           <div className="home-callout__badge">
             {card.BADGE}
           </div>
-          <h3 className="card__title is-size-h2 m-bottom-2">{card.TITLE}</h3>
+          <h3 className="card__title is-size-h2 m-bottom-2">
+            {card.TITLE}
+            {(card.TAG.length > 0 ? (<span className="tag--beta">{card.TAG}</span>) : '')}
```

Comment:
This works! I think best practice here is to return `null` if you're not looking to render anything. https://reactjs.org/docs/jsx-in-depth.html#booleans-null-and-undefined-are-ignored

Reply:
Changed to return `null`